os-server/src/main/java/app/openschool/coursemanagement/controller/CourseController.java

```diff
@@ -38,9 +40,16 @@ public ResponseEntity<Map<String, List<CategoryDtoForRegistration>>> findCategor
     return ResponseEntity.ok(courseService.findCategoriesByTitle(title));
   }
 
-  @GetMapping("/courses/suggested")
+  @GetMapping("/users/{userId}/courses/suggested")
```

Comment:
please move this into the user Controller 
or create a new one UserCourseController in this package